docs/src/ref_impl/creating_appvm.md

```diff
@@ -57,9 +54,8 @@ Each VM has the following properties:
 
 | **Property** | **Type**                  | **Unique** | **Description**                                                                                               | **Example**         |
 | -------------- | --------------------------- | ------------ | --------------------------------------------------------------------------------------------------------------- | --------------------- |
-| name         | str                       | yes        | This name is prefixed with `vm-` and will be shown in microvm list. The prefixed name - e.g. `vm-chromium` will be also the VM hostname.                                     | “chromium”        |
+| name         | str                       | yes        | This name is postfixed with `-vm` and will be shown in microvm list. The name - e.g. `chromium-vm` will be also the VM hostname.                                     | “chromium”        |
```

Comment:
Here we could use more generic `browser-vm` so we don't bind ourselves and our users to a specific browser. This is already practical since @avnik 's demo apps refactoring supports also Firefox - see https://github.com/tiiuae/ghaf/pull/271/files#diff-f7c3259ad5972b770b994e080b0ac581c3a185a4d1f14086732120d7566e98d2R51

Reply:
Let's postpone naming for now.